apps/platform-app/src/app/[locale]/(profile)/(components)/CookieBanner/CookieBanner.tsx

```diff
@@ -0,0 +1,52 @@
+'use client'
+import { Button } from '@gdh/ui-system'
+import { useEffect, useState } from 'react'
+import styles from './CookieBanner.module.scss'
+
+export const CookieBanner: React.FC = () => {
+  const [isBannerVisible, setIsBannerVisible] = useState<boolean>(false)
+
+  useEffect(() => {
+    try {
+      const hasAcceptedCookies = localStorage.getItem('cookiesAccepted')
```

Comment:
**issue**: Consider adding a check for localStorage availability to enhance compatibility with environments where localStorage is not available.